strongbox-security/strongbox-security-api/src/main/java/org/carlspring/strongbox/configuration/StrongboxDelegatingPasswordEncoder.java

```diff
@@ -0,0 +1,72 @@
+package org.carlspring.strongbox.configuration;
+
+import com.hazelcast.util.Base64;
```

Comment:
Why this import? Isn't the standard class from the Java SDK good enough?

Reply:
Java SDK Base64 doesn't have an API to check if a given string is already encoded in Base64. But yeah we can go without this library. I will use SDK Base64

Reply:
I'm sure Spring must have something useful in this regard. My concern is that Hazelcast is a library for caching and, perhaps, some day, we might think of replacing it with some other solution and then this code would have to be revisited. In this particular case, I don't think that this is the correct import, so I would advise you to look for something better.